TheWarWithin/Priorities/PriestShadow.simc

```diff
@@ -138,7 +138,7 @@ actions.filler+=/shadow_word_pain,cycle_targets=1,if=set_bonus.tier31_4pc
 actions.filler+=/shadow_word_pain,cycle_targets=1,if=!set_bonus.tier31_4pc
 
 actions.empowered_filler+=/mind_spike_insanity,cycle_targets=1
```

Comment:
Please add `,if=buff.mind_spike_insanity.up` (for the same reason the Mind Flay change is needed below.